suite-native/module-add-accounts/src/hooks/useAddCoinAccount.ts

```diff
@@ -122,23 +122,8 @@ export const useAddCoinAccount = () => {
         networkSymbol: NetworkSymbol;
     }) => availableNetworkAccountTypes.get(networkSymbol) ?? [NORMAL_ACCOUNT_TYPE];
 
-    const getNetworkToAdd = ({
-        networkSymbol,
-        accountType,
-    }: {
-        networkSymbol: NetworkSymbol;
-        accountType?: AccountType;
-    }) => {
-        const type = accountType ?? NORMAL_ACCOUNT_TYPE;
-
-        return supportedNetworks.filter(
-            network =>
-                network.symbol === networkSymbol &&
-                (type === NORMAL_ACCOUNT_TYPE
-                    ? network.accountType === undefined
-                    : network.accountType === type),
-        )[0];
-    };
+    const getNetworkToAdd = ({ networkSymbol }: { networkSymbol: NetworkSymbol }) =>
+        supportedNetworks.filter(network => network.symbol === networkSymbol)[0];
```

Comment:
This gets simplified from "get network specific for the accountType"
to just getting the main network (for normal accountType),
and `accountType` is now passed alongside.